pkg/agent/agent.go

```diff
@@ -714,7 +672,7 @@ func (i *Initializer) setupGatewayInterface() error {
 		gatewayIface.OVSPortConfig = &interfacestore.OVSPortConfig{PortUUID: gwPortUUID, OFPort: gwPort}
 		i.ifaceStore.AddInterface(gatewayIface)
 	} else {
-		klog.V(2).Infof("Gateway port %s already exists on OVS bridge", i.hostGateway)
+		klog.V(2).InfoS("Gateway port already exists on OVS bridge", "name", i.hostGateway, "ofPort", gatewayIface.OFPort)
```

Comment:
The same request exists on antrea-tun0.

Reply:
I don't understand. What do you mean by "desired value" here?
If the port already exists, we do not try to change the port value to `DefaultHostGatewayOFPort`, instead we just get whatever the current value is, store it to `nodeConfig.GatewayConfig`, and use that to program OVS flows.

Reply:
My bad, I originally thought the plan is to always use the static port, e.g., DefaultHostGatewayOFPort or DefaultTunOFPort in the OpenFlow entries, then we don't need to cache the exact OpenFlow port values assigned for the interfaces in the memory, then we may need the conflict between the two values before and after upgrade. But it seems the fix is to keep the logic that using the real value of OpenFlow ports read from OVS in the flow entries, then we don't need to think about the differences.

Reply:
BTW, if my above understanding is correct, we may also need to modify this consumption of the default values of OpenFlow ports,
https://github.com/antrea-io/antrea/blob/main/pkg/agent/multicluster/mc_route_controller.go#L460
https://github.com/antrea-io/antrea/blob/main/pkg/agent/openflow/client.go#L981

Reply:
We are sticking to the current behavior, which is that if the port already exists we do not update the ofport number (unless we need to re-create the port, e.g. because the tunnel type has changed).

> BTW, if my above understanding is correct, we may also need to modify this consumption of the default values of OpenFlow ports

Thanks for catching this. The original code had a few occurrences where `DefaultTunOFPort` was used instead of `nodeConfig.TunnelOFPort`, and I missed these 2. Fixed now (see the last commit).